gravitee-am-ui/src/app/domain/settings/password-policies/password-policies-idp-select-dialog/password-policies-idp-select-dialog.component.ts

```diff
@@ -72,13 +72,18 @@ export class PasswordPoliciesIdpSelectDialogComponent {
     this.dialogRef.close(this.result);
   }
 
-  handleEvent(event: { id: string; selected: boolean }, array: string[]) {
+  handleEvent(event: { id: string; selected: boolean }, result: DialogResult): void {
     if (event.selected) {
```

Comment:
**result: DialogResult** in such case can be removed from paratemer list because its a member of this class.

maybe single list is now enough and we dont need idpsToUnlink list